pkg/virt-api/rest/subresource.go

```diff
@@ -190,6 +190,22 @@ func (app *SubresourceAPIApp) httpGetRequestHandler(request *restful.Request, re
 	response.WriteEntity(v)
 }
 
+func (app *SubresourceAPIApp) httpGetConsoleLogHandler(request *restful.Request, response *restful.Response, validate validation, getURL URLResolver) {
```

Comment:
As well as for `GetStream` in the virt-handler, we can generalize the `httpGetRequestHandler` or at least changing the name in `httpGetStreamLogHandler`